sovtokenfees/sovtokenfees/req_handlers/fees_utils.py

```diff
@@ -0,0 +1,54 @@
+from sovtokenfees.domain import build_path_for_set_fees
+
+from common.serializers.serialization import state_roots_serializer, proof_nodes_serializer, config_state_serializer
+from plenum.common.constants import BLS_LABEL, MULTI_SIGNATURE, ROOT_HASH, PROOF_NODES
+from state.trie.pruning_trie import rlp_decode
+
+
+class BatchFeesTracker:
+    def __init__(self):
+        self.fees_in_current_batch = 0
+        self._deducted_fees = {}
+
+    def add_deducted_fees(self, txn_type, seq_no, fees):
+        key = "{}#{}".format(txn_type, seq_no)
+        self._deducted_fees[key] = fees
+
+    def has_deducted_fees(self, txn_type, seq_no):
+        return "{}#{}".format(txn_type, seq_no) in self._deducted_fees
+
+
+def get_fee_from_state(state, fees_alias=None, is_committed=False, with_proof=False, bls_store=None):
```

Comment:
please, use "lookup" method for read requests and "get_from_state" for write txns instead of this helper function.

Reply:
ok